### DISCUSSION TITLE
Running Nomadnet on WSL

### DISCUSSION THREAD

**Discussion by altyee** [markqvist/Reticulum#393] [Show and tell]
I've managed to get it running on WSL2 on Windows, over a Debian distro. Installation and running are pretty much the same with native Linux. It runs out of the box. 

The only thing that took me a couple of hours to figure out is that I need to set the WSL's networking mode to "mirrored" as described here (https://learn.microsoft.com/en-us/windows/wsl/networking) for it to receive any announcements on the test-net. I guess it's due to the original NAT mode of the WSL not forwarding certain ports to the VM. I hope this will help someone on a similar setup save some time.

**Comment by markqvist** [comment]
Thanks a bunch for figuring this out and posting the instructions!
